src/target/riscv/riscv.c

```diff
@@ -3791,6 +3791,10 @@ int riscv_enumerate_triggers(struct target *target)
 				if (tdata1 & MCONTROL_DMODE(riscv_xlen(target)))
 					riscv_set_register(target, GDB_REGNO_TDATA1, 0);
 				break;
+			case 6:
```

Comment:
Looks correct to me.  (FWIW, the leading whitespace on some of the pre-existing lines within that switch statement seem to be different than the newly added lines.  At least in the way that GitHub's diff shows that area of code.  Not a big deal, obviously)